editor.planx.uk/src/pages/FlowEditor/components/Team/formSchema.tsx

```diff
@@ -5,3 +5,13 @@ export const addNewEditorFormSchema = Yup.object({
   lastName: Yup.string().required("Required"),
   email: Yup.string().email("Invalid email address").required("Required"),
 });
+
+export const editorFormSchema = Yup.object({
```

Comment:
question: can the existing schema in this file be reused (and renamed more generically)? It looks very similar, except you've got more specific error messages (which maybe I should have used in the first place!)